client/components/jetpack/scan-threats/index.tsx

```diff
@@ -51,6 +55,58 @@ const ScanError: React.FC< { site: Site } > = ( { site } ) => {
 	);
 };
 
+const getThreatCountMessage = (
+	countHighSeverity: number,
+	countLowSeverity: number,
+	prefix: string | null,
+	suffix: string | null
+) => {
+	let lowThreatsSummary = '';
+	if ( countLowSeverity ) {
+		lowThreatsSummary = translate( '%(lowCount)s low risk item', '%(lowCount)s low risk items', {
+			args: {
+				lowCount: numberFormat( countLowSeverity, 0 ),
+			},
+			comment: '$(lowCount)s is the number of low severity items found.',
+			count: countLowSeverity,
+		} );
+	}
+
+	let highThreatsSummary = '';
+	if ( countHighSeverity ) {
+		highThreatsSummary = translate( '%(threatCount)s threat', '%(threatCount)s threats', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( '%(threatCount)d threat found', '%(threatCount)d threats found' )` *ES Score: 7*